action.yml

```diff
@@ -55,8 +55,8 @@ inputs:
     required: false
 runs:
   using: docker
-  # image: Dockerfile
-  image: docker://ewjoachim/coverage-comment-action:latest
+  image: Dockerfile
+  # image: docker://ewjoachim/coverage-comment-action:latest
```

Comment:
Why ?

Reply:
Was just for testing and forgot about the PR. Sorry!